pkg/apis/aws/validation/infrastructure_test.go

```diff
@@ -374,7 +380,7 @@ var _ = Describe("InfrastructureConfig validation", func() {
 
 			Expect(errorList).To(ConsistOf(PointTo(MatchFields(IgnoreExtras, Fields{
 				"Type":  Equal(field.ErrorTypeInvalid),
-				"Field": Equal("networks.vpc"),
+				"Field": Equal("networks.vpc.cidr"),
```

Comment:
Can you also add a test that tries to change the id of the VPC?

Reply:
done